libfreerdp/core/fastpath.c

```diff
@@ -727,6 +727,39 @@ static BOOL fastpath_recv_input_event_mouse(rdpFastPath* fastpath, wStream* s, B
 	return IFCALLRESULT(TRUE, input->MouseEvent, input, pointerFlags, xPos, yPos);
 }
 
+static BOOL fastpath_recv_input_event_relmouse(rdpFastPath* fastpath, wStream* s, BYTE eventFlags)
+{
+	rdpInput* input;
+	UINT16 pointerFlags;
+	INT16 xPos;
+	INT16 yPos;
+	WINPR_ASSERT(fastpath);
+	WINPR_ASSERT(fastpath->rdp);
+	WINPR_ASSERT(fastpath->rdp->context);
+	WINPR_ASSERT(fastpath->rdp->input);
+	WINPR_ASSERT(s);
+
+	if (!Stream_CheckAndLogRequiredLength(TAG, s, 6))
+		return FALSE;
+
+	input = fastpath->rdp->input;
+
+	Stream_Read_UINT16(s, pointerFlags); /* pointerFlags (2 bytes) */
+	Stream_Read_INT16(s, xPos);          /* xPos (2 bytes) */
+	Stream_Read_INT16(s, yPos);          /* yPos (2 bytes) */
```

Comment:
There are not positions, but deltas. This should be named like in the docs: `xDelta, and `yDelta`.